src/fetch/payouts.py

```diff
@@ -504,15 +502,26 @@ def construct_payouts(
         # TODO - After CIP-20 phased in, adapt query to return `solver` like all the others
         slippage_df = slippage_df.rename(columns={"solver_address": "solver"})
 
+    reward_token = COW_TOKEN_ADDRESS
+    native_token = Address(WETH9_ADDRESS)
+    price_day = dune.period.end - timedelta(days=1)
+    converter = TokenConversion(
+        eth_to_token=lambda t: exchange_rate_atoms(
+            native_token, reward_token, price_day
+        )
+        * t,
+        token_to_eth=lambda t: exchange_rate_atoms(
+            native_token, reward_token, price_day
```

Comment:
This is wrong. It looks as if this function is not actually used anywhere. So I will just remove it.